test/features/cloudnative/init_containers.go

```diff
@@ -20,11 +20,17 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
+// AssessSampleInitContainersOldDk checks old init container name, we need it for an upgrade scenario.
+// more details about rename https://github.com/Dynatrace/dynatrace-operator/pull/4025
+func AssessSampleInitContainersOldDk(builder *features.FeatureBuilder, sampleApp *sample.App) {
```

Comment:
this will break for the next version (as it will be always `dynatrace-operator` at that point)

also the init-container name is not dependent on the `dk` so the name is a bit confusing

Reply:
yeah I agree, it's just possible solution. To make it explicit after removing version.